include/platform/mir/graphics/display_buffer.h

```diff
@@ -58,6 +58,9 @@ class DisplayBuffer
     /** The area the DisplayBuffer occupies in the virtual screen space. */
     virtual geometry::Rectangle view_area() const = 0;
 
+    /** The size in pixels of the underlying display */
+    virtual auto pixel_size() const -> geometry::Size = 0;
```

Comment:
I'm not totally sold on the name, it is ambiguous, "size of pixel" vs "size in pixels".